CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## In master
 
+**Breaking changes**:
```

Comment:
Since you're making a new release, can you update the latest version number here?

Reply:
Yes! Missed that one. :+1: